proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -30,6 +31,12 @@ message User {
   string username = 4;
   // mutable
   bool deactivated = 5;
+  // description is the user configurable description of the user.
+  string description = 6;
+  // url is the user configurable URL in the description of the user.
```

Comment:
Will this url always include the scheme, will it never include the scheme, any other constraints? Document all constraints.

Reply:
updated